core/src/main/java/org/openstack4j/api/networking/NetworkingService.java

```diff
@@ -78,9 +79,14 @@
      * @return the FwaaS service
      */
     FirewallAsService firewalls();
-    
+
     /**
      * @return the Availability Zone Service API
      */
     AvailabilityZoneService availabilityzone();
+
+    /**
+     * @return the Service Function Chain Service API
+     */
+    SFCService sfc();
```

Comment:
i would use the expanded form ServiceFunctionChainService for the class name atleast.